docs/source/pre-requisite.md

```diff
@@ -53,7 +53,7 @@ You also need to have `OBJECT_STORAGE_ENDPOINT_URL`, `OBJECT_STORAGE_BUCKET_NAME
 
 ### Operate First KFP endpoint
 
-For [Operate First][1] you can find the Kubeflow Pipeline UI at this [link](http://ml-pipeline-ui-kubeflow.apps.cnv.massopen.cloud/#/pipelines).
+For [Operate First][1] you can find the Kubeflow Pipeline UI at this [link](http://ml-pipeline-ui-kubeflow.apps.zero.massopen.cloud/#/pipelines).
```

Comment:
this service is not available

Reply:
migration is not finished yet, I see all apps https://odh-dashboard-opf-dashboard.apps.zero.massopen.cloud/ not installed

Reply:
@4n4nd what is the new endpoint for Kubeflow pipelines? Thanks!

Reply:
@pacospace The ODH dashboard is lying for some reason. We need to fix that. (https://github.com/operate-first/odh-dashboard/issues/9) The components are available.

I think you can use this url:
http://istio-ingressgateway-istio-system.apps.zero.massopen.cloud/_/pipeline/#/pipelines

Reply:
Thanks @tumido, updated it!!